programs/stake/src/stake_state.rs

```diff
@@ -1147,6 +1154,28 @@ impl<'a> StakeAccount for KeyedAccount<'a> {
     }
 }
 
+fn get_info_if_mergable(
```

Comment:
```suggestion
fn get_info_if_mergeable(
```

Reply:
I wish `set spell` worked for snake-case :slightly_frowning_face: 

I'll add this to my next round of cleanup so I can get all the callers too.  Thanks!

Reply:
I've noticed recently that Firefox spellcheck handles snake case shockingly well... so there must be some kind of analog out there somewhere.